proto/buf/alpha/wasmplugin/v1/wasmplugin.proto

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020-2023 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.wasmplugin.v1;
+
+// Meta describes buf specific extensions for a wasm plugin.  A metadata may
```

Comment:
```suggestion
// Meta describes buf specific extensions for a wasm plugin. Metadata may
```

Reply:
Changing the name to `ExecConfig` after discussing with P